dbschema/partner.esdl

```diff
@@ -0,0 +1,38 @@
+module default {
+  type Partner extending Resource, Project::ContextAware, Mixin::Pinnable, Mixin::Taggable {
+    required active: bool;
+    required globalInnovationsClient: bool;
+
+    pmcEntityCode: str {
+      constraint regexp(r'^[A-Z]{3}$');
+    }
+
+    #address: str; #TODO - this needs figured out - needed on here and Organization?
+    types: array<Partner::Type>;
+    financialReportingTypes: array<Partner::FinancialReportingType>;
+
+    pointOfContact: default::Person;
+    single languageOfWiderCommunication: default::Language;
+    required organization: default::Organization;
```

Comment:
We should probably create this automatically with a trigger. It can be given the same project context for this object.
Same as this
https://github.com/SeedCompany/cord-api-v3/blob/9f47163ab06438265a7e8dff549522ab74bcc596/dbschema/language.esdl#L26-L32
Though there's no `ownSensitivity` here.


I wonder if this should be exclusive too at least for now, so it's a 1-1 relationship.

Reply:
I'm not following here.  In the case of your example I understand that we want to create an `Ethnologue` based on a new `Language`.  Are we saying here that we want to automatically create a `Partner` when we create an `Organization`?

Reply:
After a Slack discussion, the solution here for now is to do this:

```
required organization: Organization {
      constraint exclusive;
    };
```

which denotes a 1-to-1 relationship between `Partner` and `Organization`.